utils/webapp_connector.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
@lieut-data, you had previously suggested that this could be owned by the Desktop App. Were you thinking it was okay for this to not even exist unless the Desktop App loaded the Webapp and then just wrap any calls to it from the Webapp in a conditional before trying to use?

Reply:
0/5 on the past suggestion -- it's nice to have the connector be permanently present and avoid inline conditionals, so this seems fine.